backend/Converters/ContactConverter.cs

```diff
@@ -0,0 +1,26 @@
+using DTO;
+using DefaultNamespace;
```

Comment:
fix DefaultNamespaces in the PR to its folder's namespace

Reply:
I already change that.